src/components/GoogleLogin.vue

```diff
@@ -1,8 +1,15 @@
 <script lang="ts" setup>
-const callback = (response: Response) => {
+import { post } from "../utils";
+interface IGetUserAuthInfoResponse extends Response {
+  credential: string;
+}
+const callback = (response: IGetUserAuthInfoResponse) => {
```

Comment:
there's already a type in the vue3-google-login package for this fyi

https://yobaji.github.io/vue3-google-login/#typescript

Reply:
ok this should be fixed in 203e7f9